docs/dev-tools/zombienet/index.md

```diff
@@ -0,0 +1,600 @@
+|  `help`   |                                            Prints help information                                            |                                                                                                                                                          -                                                                                                                                                           |
+
+
+!!! warning
+    For the `spawn` command to work on macOS, users need to be aware that the Polkadot binary is currently not compatible with macOS. As a result, macOS users will need to clone the [Polkadot repository](https://github.com/paritytech/polkadot-sdk){target=_blanket}, generate a release, and manually add it to their PATH.
+
+Then, you can use the following flags to customize the behavior of the CLI:
+
+|                   Flag                    |                                                Description                                                |
+| :---------------------------------------: | :-------------------------------------------------------------------------------------------------------: |
+|            `-p`, `--provider`             |  Override provider to use (choices: `podman`, `default`, and, `native`). By default it uses `kubernetes`  |
+|           `-d`, `--dir` <path>            | Directory path for placing the network files instead of random temp one (e.g. -d /home/user/my-zombienet) |
+|              `-f`, `--force`              |                                    Force override all prompt commands                                     |
+|        `-l`, `--logType` <logType>        |   Type of logging on the console (choices: `table`, `text`, and, `silent`). By default it uses `table`    |
+|             `-m`, `--monitor`             |                              Start as monitor, do not auto clean up network                               |
+| `-c`, `--spawn-concurrency` <concurrency> |                   Number of concurrent spawning process to launch. By default it is `1`                   |
+|              `-h`, `--help`               |                                         Display help for command                                          |
+
+
+## Configuration Files {#configuration-files}
+
+The network configuration can be given in either `json` or `toml` format. Zombienet repository also provides a [folder with some examples](){target=_blanket} of configuration files that can be used as a reference.
+
+!!! note
+    Each section may include provider-specific keys that are not recognized by other providers. For example, if you use the native provider, any references to images for nodes will be disregarded.
+
+### Settings
```

Comment:
I'm not sure if the description column should have a period at the end of the sentence

Reply:
Hey @eshaben, could you please confirm this for me? I'm not sure either. 🙏 Thanks!

Reply:
@nhussein11 no, it doesn't need the period